### DISCUSSION TITLE
[Question]: Only publish modified extensions AL-Go

### DISCUSSION THREAD

**Discussion by DesarrolloBC** [microsoft/AL-Go#1586] [Q&A] [answered]
### Question

Hello,

I've been searching for a way to publish only the modified extensions in the commits from the latest fetch on AL-Go.

I didn't find any setting in AL-Go to specify that only the modified extensions should be published. Why am I asking this?

Because I've found that deployment is much faster when only the modified extensions are deployed. Also, I've tried the combination of the InstallApps setting with AppFolders, and somehow the processing time and the average number of errors have been drastically reduced.

However, it's not practical to be adjusting these settings for every scheduled deployment. That's why I'm asking if there's a way to deploy only the modified extensions to BC Online.

Thanks for your amazing work.

Best regards,
Antonio

**Comment by freddydk** [comment, marked as answer]
The latest version of AL-Go for GitHub supports incremental builds.
By setting that up, AL-Go will grab the un-modified apps from the last known good build and as such not re-deploy them as they are already deployed.
See https://aka.ms/algosettings#incrementalbuilds
